update.sh/target-pipeline.groovy

```diff
@@ -160,6 +160,13 @@ node {
 					git commit -m "Update to $version" || true
 				done
 
+				if git add .travis.yml; then
+					git commit -m 'Update travis.yml' || true
+				fi
+
+				git add -A
```

Comment:
This is not something that would be acceptable here.  We only add the changes that we explicitly look for to prevent surprises.  I'd rather not have a misbehaving script adding a commit that deletes useful things (ex. curl gets an incomplete list that is still an HTTP 200) or adding unrelated file changes.

Reply:
would you be fine with only the first commit?

Reply:
> This is not something that would be acceptable here. We only add the changes that we explicitly look for to prevent surprises. I'd rather not have a misbehaving script adding a commit that deletes useful things (ex. curl gets an incomplete list that is still an HTTP 200) or adding unrelated file changes.

@yosifkit all update scripts i have looked at only overwrite files, so even if curl returns an incomplete list, it shouldn't delete any files. (except the nextcloud one)

Is there any chance that you merge a similar change, that more explicitly adds the images that should be removed? Or do you simply want Jenkins to only add or update files?